lib/embulk/input/marketo/timeslice.rb

```diff
@@ -39,6 +88,7 @@ def transaction(config, &control)
               encryption_key: config.param(:encryption_key, :string),
               from_datetime: from_datetime,
               to_datetime: to_datetime,
+              workers: config.param(:workers, :integer, default: 5), # prime number is better for shuffling fetch time range in each task
```

Comment:
Yes. Local executor decides the # of worker threads as following:
https://github.com/embulk/embulk/blob/master/embulk-core/src/main/java/org/embulk/exec/LocalThreadExecutor.java#L19